docs/content/drafts/PageLayout.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: PageLayout
+status: Draft
+description: TODO
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<PageLayout>
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### With dividers
+
+```jsx
+<PageLayout>
+  <PageLayout.Header divider="line">Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane divider="line">Pane</PageLayout.Pane>
+  <PageLayout.Footer divider="line">Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### Condensed
+
+```jsx
+<PageLayout columnGap="condensed" rowGap="condensed">
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+## Props
+
+### PageLayout
+
+<!-- TODO: Resoponsive variants -->
+
+<PropsTable>
+  <PropsTableRow
+    name="containerWidth"
+    type={`| 'full'
+| 'medium'
+| 'large'
+| 'xlarge'`}
+    defaultValue="'full'"
+    description="Define the maximum width of the page container."
+  />
+  <PropsTableRow
+    name="outerSpacing"
+    type={`| 'normal'
+| 'condensed'`}
+    defaultValue="'normal'"
+  />
+  <PropsTableRow
+    name="columnGap"
+    type={`| 'normal'
+| 'condensed'`}
+    defaultValue="'normal'"
+  />
+  <PropsTableRow
+    name="rowGap"
+    type={`| 'normal'
+| 'condensed'`}
+    defaultValue="'normal'"
+  />
+  <PropsTableSxRow />
+</PropsTable>
+
+### PageLayout.Header
+
+<PropsTable>
+  <PropsTableRow name="divider" type="'none' | 'line'" defaultValue="'none'" />
+  <PropsTableRow
+    name="dividerWhenNarrow"
+    type={`| 'inherit'
+| 'none'
+| 'line'
+| 'filled'`}
+    defaultValue="'inherit'"
+  />
+  <PropsTableSxRow />
+</PropsTable>
```

Comment:
Would be super nice to combine the 2 divider props 🤔 

suggestions:
```jsx
<PageLayout.Header divider="line">Header</PageLayout.Header>

<PageLayout.Header divider={{default: 'line', narrow: 'filled'}}>Header</PageLayout.Header>

// or wide? is wide the correct term here?
<PageLayout.Header divider={{wide: 'line', narrow: 'filled'}}>Header</PageLayout.Header> 

// if we want to be more explicit
<PageLayout.Header dividerVariant="line">Header</PageLayout.Header> 
<PageLayout.Header dividerVariant={{wide: 'none', narrow: 'line'}}>Header</PageLayout.Header> 
```

Reply:
@siddharthkp `filled` is only acceptable in Narrow viewports, and I'm hoping we won't have to make a distinction between Medium and Wide viewports — would that change your suggestion in any way?

Reply:
> filled is only acceptable in Narrow viewports, 

Yep, I think that's totally fair and the types for value would be the same in either API choice


>  I'm hoping we won't have to make a distinction between Medium and Wide viewports 

ooh that makes me more confident about my suggestion because this code looks pretty :)

```jsx
// always line
<PageLayout.Header divider="line">Header</PageLayout.Header> 
// switch to object syntax for line in wide + filled in narrow
<PageLayout.Header divider={{wide: 'line', narrow: 'filled'}}>Header</PageLayout.Header> 
```

Reply:
I think I'm going to move forward with the `WhenNarrow` props for now but I'd love to keep exploring these responsive prop patterns!